app/controllers/plans_controller.rb

```diff
@@ -237,9 +236,6 @@ def destroy
   def status
     @plan = Plan.find(params[:id])
     authorize @plan
-    respond_to do |format|
-      format.json { render json: @plan.status }
```

Comment:
Removed this block. Endpoint doesn't seem to be used at all.

Reply:
If the endpoint isnt removed, could we remove the route/action alltogether?

Reply:
Hmm. I think this action can probably be removed. There used to be a separate ajax call to update the plan status but I think it was refactored and became part of the json response returned back by the update/create actions of the answers_controller